duckduckgo/iOS
# Move all asset catalog colors into deprecated folders

Task/Issue URL: https://app.asana.com/0/0/1204325589959385/f
Tech Design URL:
CC:

**Description**:
Literally only does asset catalog organisation, should be no code or behaviour changes.



**Steps to test this PR**:
1. Build and run and make sure nothing is messed up. Make sure widgets look okay since they use an asset catalog



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
